test/Microsoft.Azure.WebJobs.Host.EndToEndTests/Scale/ScaleHostEndToEndTests.cs

```diff
@@ -86,28 +94,22 @@ public async Task ScaleManager_GetScaleStatusAsync_ReturnsExpected(bool tbsEnabl
                 services.AddSingleton<IConcurrencyStatusRepository, TestConcurrencyStatusRepository>();
             })
             .ConfigureWebJobsScale((context, builder) =>
-                {
-                    builder.UseHostId(hostId);
-                },
-                scaleOptions =>
-                {
-                    scaleOptions.IsTargetScalingEnabled = tbsEnabled;
-                    scaleOptions.MetricsPurgeEnabled = false;
-                    scaleOptions.ScaleMetricsMaxAge = TimeSpan.FromMinutes(4);
-                    scaleOptions.IsRuntimeScalingEnabled = true;
-                });
+            {
+                builder.UseHostId(hostId);
 
-            foreach (TriggerMetadata t in triggerMetadata)
+                Assembly assembly = Assembly.GetExecutingAssembly();
```

Comment:
This isn't needed - DefaultStartupTypeLocator has a default ctor you can use

Reply:
The DefaultStartupTypeLocator  ctor returns `testhost` assembly but we need `Microsoft.Azure.WebJobs.Host.EndToEndTests`:
![image](https://github.com/Azure/azure-webjobs-sdk/assets/2578359/80dd18a0-c446-4dd7-92a8-2be06980e443)